testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/statistics/AbstractDataSourcePoolStatisticsTestCase.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ * Copyright ${YEAR} Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.jboss.as.test.integration.jca.statistics;
+
+import jakarta.annotation.Resource;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.client.helpers.Operations;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.dmr.ModelNode;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
WildFly import ordering style is:

import static

import java
import javax

import other

Reply:
Note there are many files that violate this, including some you've refactored. Those many files are why we can't enforce it via codestyle. :(

Reply:
fixed